### PR TITLE
Filters for dateCreated and dateModified

### PR DESCRIPTION
**Pull request by fgalan**
@kzangeli 
@crbrox

### CODE REVIEW

File: src/lib/mongoBackend/MongoGlobal.cpp [hunk 1]
```diff
@@ -1049,16 +1049,7 @@ static bool matchFilter
 {
   /* First, look for unary operators */
   if ((opr == OPR_EXIST) || (opr == OPR_NOT_EXIST))
-  {
```
Comment (fgalan) [inline src/lib/mongoBackend/MongoGlobal.cpp]: Not really part of this PR, but a bit of cleanup regarding `+type` removal.

NTC

File: src/lib/mongoBackend/MongoGlobal.cpp [hunk 2]
```diff
@@ -1609,43 +1612,18 @@ static bool addBsonFilter
   }
   else if (opr == OPR_EXIST)
   {
-    if (std::string(right) == ENT_ENTITY_TYPE)
-    {
-      // Special case: entity type
```
Comment (fgalan) [inline src/lib/mongoBackend/MongoGlobal.cpp]: Not really part of this PR, but a bit of cleanup regarding `+type` removal.

NTC

File: src/lib/mongoBackend/MongoGlobal.cpp [hunk 3]
```diff
@@ -1609,43 +1612,18 @@ static bool addBsonFilter
...
-    }
+    bb.append("$exists", true);
+    bob.append(k, bb.obj());
+    f = bob.obj();
   }
   else if (opr == OPR_NOT_EXIST)
   {
-    if (std::string(right) == ENT_ENTITY_TYPE)
-    {
-      // Special case: entity type
```
Comment (fgalan) [inline src/lib/mongoBackend/MongoGlobal.cpp]: Not really part of this PR, but a bit of cleanup regarding `+type` removal.

NTC

**Comment by crbrox** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM
